cmd/manager/main.go

```diff
@@ -105,8 +105,7 @@ func main() {
 
 	// Setup the notifier if it is enabled
 	if *messagingProviderName != "" {
-		notifier, err = notifierbuilder.BuildNotifier(*messagingProviderName)
-		if err != nil {
+		if notifier, err = notifierbuilder.BuildNotifier(*messagingProviderName); err != nil {
```

Comment:
Hmm not sure about this simplification - even though it works, it might be a little confusing.

If you do `a, err := someFunction(); err != nil {`, `a` and `err` are scoped to that if statement. But if the variables exist and you do `a, err = someFunction(); err != nil {`, (note the `=`) they won't be scoped to the if statement.

Reply:
Sure, I didn't think about how it could cause confusion. In this case I didn't want to scope the the if statement. I'll revert this.